core/templatetags/discover_uni_tags.py

```diff
@@ -59,17 +59,47 @@ def map_distance_learning_values(key, language):
 
 
 @register.simple_tag
-def should_show_accordion(course, accordion_type):
+def should_show_accordion(courses, accordion_type):
```

Comment:
Could consider doing something like this as an alternative? (this is for future ref though Matt, I know you don't have time atm)
```python
def should_show_accordion(courses, accordion_type):
    attr_name_lookup = {
            STUDENT_SATISFACTION_KEY: "show_statisfaction_stats",
            # ...
            }
    attr_name = attr_name_lookup[accordion_type]
    if type(courses) == tuple:
        show = False
        for course in courses:
            if getattr(course, attr_name):
                show = True
        return show
    return getattr(courses, attr_name)
```